### PR TITLE
completed css-three-column-layout

### PR DESCRIPTION
**Pull request by ellie-hwang**
css-three-column

### CODE REVIEW

File: css-three-column-layout/styles.css [hunk 1]
```diff
@@ -0,0 +1,64 @@
...
+
+.row {
+  display: flex;
+  justify-content: space-between;
+}
+
+.column-full {
+  width: 100%;
+  text-align: center;
+}
```
Comment (codingcodymiller) [inline css-three-column-layout/styles.css:38]: Column classes should only include `flex-basis` or `width`, anything else should be a separate class. You need to keep your layout classes untainted and reusable.

File: css-three-column-layout/styles.css [hunk 2]
```diff
@@ -0,0 +1,64 @@
...
+
+.column-one-third {
+  text-align: center;
+  flex-basis: 33%;
+}
```
Comment (codingcodymiller) [inline css-three-column-layout/styles.css:43]: Column classes should only include `flex-basis` or `width`, anything else should be a separate class. You need to keep your layout classes untainted and reusable.

File: css-three-column-layout/styles.css [hunk 3]
```diff
@@ -0,0 +1,64 @@
...
+
+.container {
+  width: 900px;
+  margin: 50px auto 0;
+}
+
+.row {
+  display: flex;
+  justify-content: space-between;
+}
```
Comment (codingcodymiller) [inline css-three-column-layout/styles.css:33]: The `row` class should only be `display: flex`, anything else should be a separate class. You need to keep your layout classes untainted and reusable.

**Review by codingcodymiller — CHANGES_REQUESTED** [review]

File: css-three-column-layout/styles.css (hunk 3 above)
Comment (ellie-hwang) [reply css-three-column-layout/styles.css]: Fixed the .row and .column classes 
css-three-column-layout-02

**Review by codingcodymiller — APPROVED** [review]
Nice work!!

**Pull request merged**